### PR TITLE
SEM - 046: Input Variable Formulario

### PR DESCRIPTION
**Pull request by Victoria-Jimenez-Martinez**
Como usuario quiero ver un formulario para ingresar información diferente.

### CODE REVIEW

**Comment by DanX3N0** [comment]
figma:
image
pr:
image
image
image
image
image

it's according to figma

File: src/ui/utils/hooks/useForm.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  const handlerChangeForm = ({ target }: ChangeEvent<HTMLInputElement>) => {
+    setForm({ ...form, [target.name]: target.value });
+  };
+
+  const handlerResetForm = () => {
+    setForm(initForm);
+  };
+
+  return { form, handlerChangeForm, handlerResetForm };
+};
```
Comment (Ditmar) [inline src/ui/utils/hooks/useForm.ts:24]: Nice transformation i realyy appreciate the work with the generic type of data

**Review by Ditmar — APPROVED** [review]
nice work!

File: src/assets/icons/ojo.svg
Comment (Ditmar) [inline src/assets/icons/ojo.svg:1]: please only this could you change the name of the SVG to eye.svg

**Review by Ditmar — CHANGES_REQUESTED** [review]

**Review by Ditmar — APPROVED** [review]

**Pull request closed without merging**